mealie/db/models/group/group.py

```diff
@@ -67,7 +73,7 @@ class Group(SqlAlchemyBase, BaseMixins):
     webhooks: Mapped[list[GroupWebhooksModel]] = orm.relationship(GroupWebhooksModel, **common_args)
     recipe_actions: Mapped[list["GroupRecipeAction"]] = orm.relationship("GroupRecipeAction", **common_args)
     cookbooks: Mapped[list[CookBook]] = orm.relationship(CookBook, **common_args)
-    server_tasks: Mapped[list[ServerTaskModel]] = orm.relationship(ServerTaskModel, **common_args)
+    server_tasks: Mapped[list["ServerTaskModel"]] = orm.relationship("ServerTaskModel", **common_args)
```

Comment:
Why did this need to change?
(I don't have a problem with it, just not sure why the change is being made)

Reply:
Ah yeah, that's because I removed the line by accident then put it back (and accidentally did it slightly differently). Specifically it's different because of how imports work with SQLAlchemy: I could've put it back how it was, but I also moved the import to typing only (which means it's not actually imported at runtime):

```python
# earlier in the file
if TYPE_CHECKING:
    ...
    from ..server.task import ServerTaskModel
```

This is a better way of handling imports in this scenario since it's slightly more performant and avoids circular imports (though that's not actually an issue here). In theory we should replace all of our SQLAlchemy imports to do this, but the performance benefit is negligible so it's not worth the effort.

---

TL;DR it's a totally unnecessary change I did by accident, but I'm inclined to leave it because it's technically better anyway

Reply:
At least we had fun 😅